### PR TITLE
ci: create CODEOWNERS

### PR DESCRIPTION
**Pull request by margaretkennedy**

### CODE REVIEW

**Comment by margaretkennedy** [comment]
Would "chore" be more appropriate?

**Comment by dsmmcken** [comment]
> Would "chore" be more appropriate?

`ci:`

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+**/docs/**.md @margaretkennedy
```
Comment (mofojed) [inline .github/CODEOWNERS:1]: You need the `**` for the directory, then `*.md` for the file.
```suggestion
**/docs/**/*.md @margaretkennedy
```

**Review by mofojed — CHANGES_REQUESTED** [review]

**Review by dsmmcken — APPROVED** [review]

**Review by mofojed — DISMISSED** [review]

**Review by mattrunyon — CHANGES_REQUESTED** [review]
We should not have anything with only 1 codeowner. Puts a complete stop if Margaret is out for any reason

**Comment by dsmmcken** [comment]
That's a fair point. It's more like we wanted her to be auto-added as a reviewer for an doc change but you are right this would be a hard requirement which isn't necessarily what we want either. I am gonna recommend just closing this PR for now.

**Comment by mattrunyon** [comment]
I think you could achieve this w/ a custom action that uses `github-script` ([info](https://github.com/actions/github-script)) and `github.rest.pulls.requestReviewers` ([docs](https://octokit.github.io/rest.js/v21/#pulls-request-reviewers))

If you want I can probably put that together pretty quickly. Seems pretty straightforward

**Comment by dsmmcken** [comment]
Re-closing, we've decided we didn't want a blocking code owner as part of this repos workflow.

We did add an action though that adds you as a reviewer for any docs changes automatically. 

See:
#778

**Pull request closed without merging**